core_lib/web_helpers/helpers.py

```diff
@@ -0,0 +1,20 @@
+from core_lib.session.security_handler import SecurityHandler
+from core_lib.web_helpers.decorators import handle_exception
+import logging
+from core_lib.web_helpers.web_helprs_utils import WebHelpersUtils
+logger = logging.getLogger(__name__)
+
+
+def require_login_helper(request, policies, func, *args, **kwargs):

```

Comment:
`require_login_helper` is the name of the file `require_login` is the name of the function